pages/irac-2022.md

```diff
@@ -49,13 +47,79 @@ We have invited renowned speakers who will give an introductory overview for eac
 ![picture](/assets/images/irac-2022-boya-goa.jpg){: style="height: 100px; width: 100px; border-radius: 100px; border: 3px solid #e9ecef;" .me-3 .float-start}
 **Boya Gao** - Boya is a post-doctor at plant protection college, [Nanjing Agricultural University](http://english.njau.edu.cn/mainm.htm) who researches insect migration and radar entomology.
 
-## Abstract submission
-
-[Abstract submission](https://forms.gle/8B9VVcf5Mq2tERhZ8){: .btn .btn-primary }
-
-<!-- ## Registration -->
+## Programme
+
+<style>
+  .table th:first-of-type {
+    width: 20%;
+  }
+</style>
+
+An optional pre-conference **bird watching excursion** to the Flüela Pass (with chance to see snowfinches) is organized by Fränzi Korner-Nievergelt. Email <irac2022@vogelwarte.ch> if you want to join.
+
+### Saturday - June 25
+
+time        | presentation
```

Comment:
`Let me know if I can copy/paste abstracts from the form spreadsheet`
Everyone answered `Yes` to the publication question on submission, so that should be fine, yes.